src/main/java/com/cloudbees/jenkins/support/impl/GCLogs.java

```diff
@@ -101,11 +101,13 @@ public void addContents(@NonNull Container result) {
      * @see https://bugs.openjdk.java.net/browse/JDK-7164841
      */
     private void handleRotatedLogs(@Nonnull final String gcLogFileLocation, Container result) {
+        File gcLogFile = new File(gcLogFileLocation);
```

Comment:
I'm :bee: with this since it does seem to improve the current situation, and I'm no java.io/java.nio expert, but it _seems_ to me this may have been a use case more dedicated to using `java.nio.file.Path`.